src/django/pfb_analysis/serializers.py

```diff
@@ -131,3 +136,29 @@ class Meta:
         model = AnalysisScoreMetadata
         fields = ('name', 'label', 'category', 'description',)
         read_only_fields = ('name', 'label', 'category', 'description',)
+
+
+class AnalysisLocalUploadTaskSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = AnalysisLocalUploadTask
+        fields = ('uuid', 'created_at', 'modified_at', 'created_by', 'modified_by',
+                  'status', 'job', 'upload_results_url', 'error',)
+        read_only_fields = ('uuid', 'created_at', 'modified_at', 'status', 'error',)
+
+
+class AnalysisLocalUploadTaskCreateSerializer(serializers.ModelSerializer):
+
+    neighborhood = serializers.UUIDField(write_only=True)
+
+    def create(self, validated_data):
+        validated_data.pop('neighborhood')
+        return super(AnalysisLocalUploadTaskCreateSerializer, self).create(validated_data)
+
+    class Meta:
+        model = AnalysisLocalUploadTask
+        fields = ('uuid', 'created_at', 'modified_at', 'created_by', 'modified_by',
+                  'neighborhood', 'upload_results_url',
+                  'status', 'job', 'error',)
```

Comment:
No reason other than I made too many changes to this during the time I spent correctly implementing the view/serializer. I'll reorder so that it makes more sense.